lib/util.ex

```diff
@@ -20,10 +20,16 @@ defmodule Util do
         tourneys = 1..10
                    |> Enum.map(&get_tourney_page(auth, &1))
                    |> List.flatten
-        {:ok, tourneys}
+        {:ok, %{tournament_data: tourneys, metadata: generate_tournaments_metadata()}}
     end
   end
 
+  def generate_tournaments_metadata do
+    %{
+      updated_at: DateTime.now("Etc/UTC") |> elem(1) |> DateTime.to_unix
```

Comment:
Since we're not checking for an error anyway, I think `DateTime.now!/` would be a better fit, and then we can get rid of the `elem(1)`.

Reply:
Makes sense!